pkg/common/constants/constants.go

```diff
@@ -36,8 +36,10 @@ const DomainYuniKornInternal = siCommon.DomainYuniKornInternal
 
 // Application
 const LabelApp = "app"
+const CanonicalLabelApplicationID = DomainYuniKorn + "app-id"
```

Comment:
Could we move `CanonicalLabelApplicationID` to line#42 (after `AnnotationApplicationID`)?

Reply:
Done